Client/Frontend/Browser/SessionData.swift

```diff
@@ -29,7 +52,7 @@ class SessionData: NSObject, NSCoding {
     **/
     init(currentPage: Int, urls: [URL], lastUsedTime: Timestamp) {
         self.currentPage = currentPage
-        self.urls = urls
+        self.urls = migrate(urls: urls)
```

Comment:
its too bad there's not a way to only check for this and do this once... its pretty lightweight though so i guess its no harm